single_joint_actuator:
    joints_update:
      type: joint_state_controller/JointStateController
      publish_rate: 50 #50

    # Position Controllers ---------------------------------------
    joint1_position_controller:
      type: effort_controllers/JointPositionController
      joint: joint1
      pid: {p: 140.0, i: 120.0, d: 10.0, i_clamp_min: -130.0, i_clamp_max: 130, antiwindup: True}
      
    # Velocity Controllers ---------------------------------------
    joint1_velocity_controller:
      type: effort_controllers/JointVelocityController
      joint: joint1
      pid: {p: 100.0, i: 100.0, d: 0.0, i_clamp_min: -255.0, i_clamp_max: 255, antiwindup: True}

    # Position Controllers ---------------------------------------
    joint2_position_controller:
      type: effort_controllers/JointPositionController
      joint: joint2
      pid: {p: 140.0, i: 120.0, d: 10.0, i_clamp_min: -130.0, i_clamp_max: 130, antiwindup: True}
      
    # Velocity Controllers ---------------------------------------
    joint2_velocity_controller:
      type: effort_controllers/JointVelocityController
      joint: joint2
      pid: {p: 265.0, i: 15.0, d: 15.0, i_clamp_min: -255.0, i_clamp_max: 255, antiwindup: True}

